### DISCUSSION TITLE
Porting my project code that running GIF from Adafruit_GFX to Arduino_GFX, functions replacing help

### DISCUSSION THREAD

**Discussion by yankobogdan** [moononournation/Arduino_GFX#72] [Q&A] [answered]
Hi. I am porting my project code that running GIF from Adafruit_GFX to Arduino_GFX. 
I need some help to rewrite Arduino_GFX (actually Adafruit_SPITFT) functions replacing
I have the following lines of code that worked fine with Adafruit lib, how can I replace them with API provided in Arduino_GFX?

```
    tft.setAddrWindow(pDraw->iX, y, iWidth, 1);     //Adafruit_SPITFT
    tft.writePixels(usTemp, iWidth, false, false);  //Adafruit_SPITFT
```
 
Thanks

**Comment by moononournation** [comment, marked as answer]
the above is Arduino_GFX class function, you can change the declaration from Arduino_GFX to Arduino_TFT for calling the function:
```
//Arduino_GFX *gfx = ...
Arduino_TFT *gfx = ...
...
gfx->setAddrWindow(...
gfx->writePixels(...
```

**Reply by yankobogdan** [reply]
Thanks for the reply. But I need to use your lib (Arduino_GFX) because of hardware setup
So I am asking if you can advise alternatives how I can replace those two Adafruit_GFX functions (setAddrWindow and writePixels) to some functions that existing in your lib

**Reply by moononournation** [reply]
same function name, simply try the above changes
